.vscode/settings.json

```diff
@@ -12,8 +12,8 @@
   "[python]": {
     "editor.formatOnPaste": false,
     "editor.codeActionsOnSave": {
-      "source.fixAll.ruff": true,
-      "source.organizeImports.ruff": true
+      "source.fixAll.ruff": "explicit",
+      "source.organizeImports.ruff": "explicit"
```

Comment:
TIL these are functionally equivalent, and the string value is preferred in later VS Code versions.
https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto

Reply:
Oh those files weren't supposed to make it into the PR. Only the changes to Home.tsx. Gonna delete the changes